lib/streamlit/__init__.py

```diff
@@ -51,7 +51,10 @@
 _LOGGER = _logger.get_logger("root")
 
 # Give the package a version.
-from importlib_metadata import version as _version
+try:
+    from importlib.metadata import version as _version
+except:
```

Comment:
I think it would be better to make a version check here instead of a blank 	`except`, e.g.:
```python
if sys.version_info < (3, 8):
    from importlib_metadata import version as _version
else:
    from importlib.metadata import version as _version
```